backend/SC4SNMP_UI_backend/common/conversions.py

```diff
@@ -62,26 +83,48 @@ def _backend2ui_map(self, document: dict, **kwargs):
                 new_vb = {
                     "family": vb[0],
                     "category": vb[1] if len(vb) >= 2 else "",
-                    "index": str(vb[2]) if len(vb) == 3 else "",
+                    "index": '.'.join(vb[2:]) if len(vb) >= 3 else "",
                 }
                 var_binds.append(new_vb)
 
             if "condition" in document[profile_name]:
                 backend_condition = document[profile_name]["condition"]
-                condition_type = backend_condition["type"]
-                field = backend_condition["field"] if condition_type == "field" else ""
+                condition_type = self.__backend2ui_profile_types[backend_condition["type"]]
```

Comment:
we have validation for `type`, right? here we will always have the correct one?

Reply:
Type is chosen in the UI from the dropdown list defined by us, so it should always be correct.